html/statbrowser.js

```diff
@@ -15,9 +15,9 @@ if (!String.prototype.trim) {
 }
 
 // Status panel implementation ------------------------------------------------
-var status_tab_parts = ["Loading..."];
```

Comment:
you should document this feature of the statpanel somewhere so it's more clear to people adding to it

Reply:
(for clarity somewhere im dm code, tho idk what would work best, maybe the statpanel proc itself? I'm a bit uncomfortable with this solution but I can't think of anything better)